docs/guides/reference/ard-expanded-processing-extent.md

```diff
@@ -2,17 +2,22 @@
 
 DEA produces surface reflectance products from Landsat and Sentinel-2 satellite observations which cover the continent and some near-shore islands of Australia. This data is used to produce derivative products such as DEA Coastlines, Water Observations, and Fractional Cover. 
 
-We are extending our baseline satellite data coverage to include Christmas Island, Cocos (Keeling) Islands, Coral Sea Islands, Heard and McDonald Islands, Norfolk Island, and islands in the Torres Strait. Additionally, we are processing imagery over shallow ocean areas to support bathymetry and oceanographic science.
+We are extending our baseline satellite data coverage to include Christmas Island, Cocos (Keeling) Islands, Coral Sea Islands, the Heard and McDonald Islands, Norfolk Island, and islands in the Torres Strait. Additionally, we are processing imagery over shallow ocean areas to support bathymetry and oceanographic science.

```

Comment:
It's the "Territory of Heard Island and McDonald Islands", so the original wording here is fine.

```suggestion
We are extending our baseline satellite data coverage to include Christmas Island, Cocos (Keeling) Islands, Coral Sea Islands, Heard and McDonald Islands, Norfolk Island, and islands of the Torres Strait. Additionally, we are processing imagery over shallow ocean areas to support bathymetry and oceanographic science.

```

Reply:
Thanks Robbi, I really appreciate the second set of eyes over this document.